int16 servo0
int16 servo1
int16 servo2